communitiesuk/funding-service-design-assessment
# Fs 2988 NSTF Dashboard

### Change description
Updating the assessor dashboard to show different columns for nstf, as per latest prototype https://fsd-pre-award.herokuapp.com/nstf-assessment/dashboard

Refactored the dashboard and search code to allow for different columns coming from assessment_store, and for searching/filtering on different fields.



### How to test
From the assessment tool dashboard, navigate into COF and NSTF and see different columns and different search options


### Screenshots of UI changes (if applicable)
![Screenshot 2023-06-27 at 09 52 03](https://github.com/communitiesuk/funding-service-design-assessment/assets/1729216/0b41a953-b5f5-467d-a3e8-d04112b582ac)
